lib/pure/collections/sequtils.nim

```diff
@@ -635,6 +635,20 @@ template mapIt*(s, typ, op: untyped): untyped =
     result.add(op)
   result
 
+macro evalOnce(exp: untyped, varName: static[string]): untyped =
```

Comment:
Why is ``varName`` a string?

Reply:
Because `genSym` likes strings? I can probably use a `untyped` argument and then stringify it if you don't like the string thing.

Reply:
Yeah, please do that.

Reply:
Done, the bootrstrapping is still hosed due to the missing `symKind`.